source/includes/markdown/_13-platform-ui.html.md

```diff
@@ -1,12 +1,21 @@
 <hr class="platform-ui-alpha">
 <section class="platform-ui-alpha">
 
-# App Components Beta
+# Overview of App Components
 
-<span class="beta-indicator">BETA</span>
+<span class="beta-indicator">BETA</span> - For access, please see [Overview of App Components](/docs/overview-of-app-components)
```

Comment:
This one seems redundant since it just links to this exact section?

Reply:
Good call! May have gone too far with copypasta for that element lol. Will fix.